src/com/edlplan/ui/fragment/InGameSettingMenu.kt

```diff
@@ -213,34 +190,19 @@ class InGameSettingMenu : BaseFragment() {
                     progress: Int,
                     fromUser: Boolean
                 ) {
-                    findViewById<TextView>(R.id.brightPreviewText)!!.text = progress.toString()
                     findViewById<TextView>(R.id.bgBrightnessText)!!.text = "$progress%"
-                    val clamped = FMath.clamp(255 * (progress / 100f), 0f, 255f).roundToInt()
-                    findViewById<View>(R.id.brightnessPreview)!!.setBackgroundColor(
-                        0xFF shl 24 or (clamped shl 16) or (clamped shl 8) or clamped
-                    )
                 }
```

Comment:
Is there a reason for removing the brightness preview? A "redesign" should not remove features.

Reply:
That was not a preview.